app/src/main/res/layout-land/profile_chooser_profile_view.xml

```diff
@@ -61,26 +61,20 @@
 
         <TextView
           android:id="@+id/profile_last_visited"
+          style="@style/SubtitleLight1"
```

Comment:
Shouldn't this be a 'Center'? Ditto for other profile_chooser_profile_view layouts, and for profile_is_admin_text below.

Reply:
No, as the gravity is handled conditionally.